src/AnnotationThread.js

```diff
@@ -419,8 +419,10 @@ class AnnotationThread extends EventEmitter {
         }
 
         this.renderAnnotationPopover = this.renderAnnotationPopover.bind(this);
+        this.handleBlur = this.handleBlur.bind(this);
```

Comment:
Minor: Should probably be in the constructor.

Reply:
👍